lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -265,15 +266,13 @@ bool LayerTreeItem::sublayerOfShared() const
     return false;
 }
 
-bool LayerTreeItem::isReadOnly() const
-{
-    return (_isSharedLayer || (_layer && !_layer->PermissionToEdit()));
-}
+bool LayerTreeItem::isReadOnly() const { return (_isSharedLayer); }
```

Comment:
Note: By design some of the functionalities should still be available if a layer has no permission to edit.